pkg/operator/ceph/csi/spec.go

```diff
@@ -316,6 +317,10 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		tp.RBDPluginUpdateStrategy = rollingUpdate
 	}
 
+	if strings.EqualFold(k8sutil.GetValue(r.opConfig.Parameters, "CSI_PLUGIN_ENABLE_SELINUX_HOST_MOUNT", "true"), "false") {
+		tp.EnableSelinuxHostMount = false
```

Comment:
If we default to false unless the setting equals `true`, I think we need to switch this:
```suggestion
		tp.EnableSelinuxHostMount = true
```

Reply:
done.